### PR TITLE
chore: Switch from MUI Icons to DHIS2 UI Icons (DHIS2-10534)

### PR DESCRIPTION
**Pull request by turban**
Fixes: https://jira.dhis2.org/browse/DHIS2-10534

This PR switches from Material UI icons to DHIS2 UI icons. I've also changed import statements for core maps components resulting in a large number of files changed (but should be quick to review). 

One drag icon was missing un UI icons, and was added to maps repo (src/components/core/icons.js) temporarily. 

The app crash page (which should of course never be visible...) now looks like this:

Screenshot 2021-02-23 at 14 29 17

Before: 
Screenshot 2021-02-23 at 14 37 56

Error message in download dialog after this PR: 

Screenshot 2021-02-23 at 14 31 32

Before: 
Screenshot 2021-02-23 at 14 34 24

After this PR: 

Screenshot 2021-02-23 at 14 03 15

Screenshot 2021-02-23 at 14 04 02

Screenshot 2021-02-23 at 14 04 43

Screenshot 2021-02-23 at 14 07 00

Screenshot 2021-02-23 at 14 07 48

Screenshot 2021-02-23 at 14 09 04

Screenshot 2021-02-23 at 14 09 50

Screenshot 2021-02-23 at 14 12 47

Screenshot 2021-02-23 at 14 13 29

Screenshot 2021-02-23 at 14 13 37

Screenshot 2021-02-23 at 14 14 41

Screenshot 2021-02-23 at 14 15 24

Screenshot 2021-02-23 at 14 16 42

Screenshot 2021-02-23 at 14 18 02

Screenshot 2021-02-23 at 14 19 00

Screenshot 2021-02-23 at 14 20 19

Screenshot 2021-02-23 at 14 21 01

### CODE REVIEW

**Review by cooper-joe — APPROVED** [review]

File: src/components/errors/styles/FatalErrorBoundary.module.css [hunk 1]
```diff
@@ -1,61 +1,34 @@
...
+    color: rgb(33, 41, 52);
+    line-height: 20px;
+    width: 500px;
 }
 
 .link {
-    font-size: 18px;
     text-decoration: underline;
     cursor: pointer;
-    margin-bottom: var(--spacers-dp24);
```
Comment (edoardo) [inline src/components/errors/styles/FatalErrorBoundary.module.css:39]: why not use the spacers?

File: src/components/core/icons.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const IconDrag = () => (
```
Comment (edoardo) [inline src/components/core/icons.js:3]: Isn't there an equivalent in `ui-icons`?

File: src/components/layers/styles/SortableHandle.module.css [hunk 3]
```diff
@@ -1,9 +1,7 @@
 .handle {
-    width: 24px;
-    height: 24px;
     position: absolute;
-    left: 6px;
+    left: 4px;
```
Comment (edoardo) [inline src/components/layers/styles/SortableHandle.module.css:3]: Use `spacers`?

**Review by edoardo — APPROVED** [review]

File: src/components/errors/styles/FatalErrorBoundary.module.css (hunk 1 above)
Comment (turban) [reply src/components/errors/styles/FatalErrorBoundary.module.css:39]: Spacers can't be used as they are defined inside FatalErrorBoundary: https://github.com/dhis2/maps-app/blob/master/src/components/app/App.js#L47

I would like to keep the error boundary there as there might also be fatal error in the CSS setup.

File: src/components/core/icons.js (hunk 2 above)
Comment (turban) [reply src/components/core/icons.js:3]: No, @cooper-joe will add one later.

File: src/components/layers/styles/SortableHandle.module.css (hunk 3 above)
Comment (turban) [reply src/components/layers/styles/SortableHandle.module.css:3]: Fixed!

**Pull request merged**